docs/compare.md

```diff
@@ -4,6 +4,7 @@
 |---|-------------|-----------|
 | [Async](#async-experience)                    | | :heavy_check_mark: |
 | [Pagination](#pagination-experience)          | | :heavy_check_mark: |
+| [Presign](#presign-experience)          | | :heavy_check_mark: |
```

Comment:
This makes my OCD sense tingling. 

Please align this =)

Reply:
sorry. Fixed